src/mpid/ch4/src/ch4_probe.h

```diff
@@ -122,9 +122,6 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_iprobe_safe(int source,
   fn_exit:
     MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_IPROBE_SAFE);
     return mpi_errno;
-
```

Comment:
I would prefer to check the return value of `mpi_errno = MPIDI_iprobe_unsafe()` instead of deleting `fn_fail`